onnxruntime/core/mlas/lib/qdwconv_kernelsize.cpp

```diff
@@ -43,13 +43,12 @@ Module Name:
 
 #include "mlasi.h"
 
-extern "C" {
-
 #if defined(MLAS_TARGET_ARM64)
 
+template<bool IsFixedPoint>
 void
 MLASCALL
-MlasConvSymDepthwiseKernelSize25ArmU8S8(
+MlasConvSymDepthwiseKernelSize25ArmS8S8Impl(
```

Comment:
This looks like a huge change. It used to be the U8S8 code is defined before S8S8, now it is reversed. it seems that this is the cause of most of the changes. Why do you need to flip the position of these two?

Reply:
it is unintentional. let me reverse them back.